zql/zql.peg

```diff
@@ -260,6 +260,20 @@ fieldExprList
       RETURN(result)
   }
 
+fieldRefDotOnly
```

Comment:
Seems like there's some duplication with fieldReference but we can tidy this up later...